crates/ra_syntax/src/ast.rs

```diff
@@ -265,90 +261,90 @@ impl<'a> Whitespace<'a> {
     }
 }
```

Comment:
how about wrapping those impl_test() into macro?

Reply:
Yeah, I'll plan to code generate them soonish.